3b2 - build errors/warnings

<!--- If you find problems or have suggestions relating to any simulator  -->
<!--- or the simh package as a whole, this is the right place to describe  -->
<!--- the problem you're having or the suggestion you wish to make: -->
<!--- Folks with general questions about the USAGE of simh simulators  -->
<!--- should start at http://simh.trailing-edge.com and once you've  -->
<!--- exhausted that, you some general google searches may be useful,  -->
<!--- and then asking on the simh mailing list. -->
<!--- NOTE: Anything between the comment delimiters that start and end this line are merely instructions to you.  They will not be visible when the issue is created.  You may remove these comments as you enter information here -->

<!--- Note that the text you enter into this comment box will be interpreted -->
<!--- as 'markdown' text.  To best leverage this you may want to tag system  -->
<!--- output and some other text as 'code' (you can click on the "styling    -->
<!--- with Markdown" link below this text box for details about markdown).   -->
<!--- Click on the Preview tab as you're creating this issue to see how the  -->
<!--- markdown message will be displayed.-->

<!--- PLEASE provide the below specified NON OPTIONAL details.
      If they are not provided, the issue will most likely be quickly 
      closed immediately.  It may be re-opened by you once you actually
      provide this information: -->


<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug or why the suggested change or enhancement would be useful -->

```
1>3b2_stddev.c
1>..\3B2\3b2_stddev.c(297) : error C2065: 'uint32_t' : undeclared identifier
1>..\3B2\3b2_stddev.c(297) : error C2146: syntax error : missing ';' before identifier 'catchup_ticks'
1>..\3B2\3b2_stddev.c(297) : error C2065: 'catchup_ticks' : undeclared identifier
1>..\3B2\3b2_stddev.c(308) : error C2065: 'catchup_ticks' : undeclared identifier
1>..\3B2\3b2_stddev.c(308) : error C2065: 'uint32_t' : undeclared identifier
1>..\3B2\3b2_stddev.c(308) : error C2146: syntax error : missing ';' before identifier 'delta'
1>..\3B2\3b2_stddev.c(308) : warning C4552: '*' : operator has no effect; expected operator with side-effect
1>..\3B2\3b2_stddev.c(311) : error C2065: 'catchup_ticks' : undeclared identifier
1>..\3B2\3b2_stddev.c(312) : error C2065: 'catchup_ticks' : undeclared identifier
```
```
1>3b2_cpu.c
1>..\3B2\3b2_cpu.c(879) : warning C4101: 'r' : unreferenced local variable
1>..\3B2\3b2_cpu.c(1035) : warning C4101: 'r' : unreferenced local variable
1>
```

--- a/3B2/3b2_cpu.c
+++ b/3B2/3b2_cpu.c
@@ -876,7 +876,6 @@ t_stat sys_boot(int32 flag, CONST char *ptr)
 {
     char gbuf[CBUFSIZE];
     size_t len = ROM_SIZE;
-    t_stat r;
 
     if ((ptr = get_sim_sw(ptr)) == NULL) {
         return SCPE_INVSW;
@@ -1032,7 +1031,6 @@ static void build_int_map()
 t_stat cpu_reset(DEVICE *dptr)
 {
     int i;
-    t_stat r;
 
     sim_debug(EXECUTE_MSG, &cpu_dev, "CPU Reset.\n");
 

--- a/3B2/3b2_stddev.c
+++ b/3B2/3b2_stddev.c
@@ -294,7 +294,7 @@ static void tod_resync(UNIT *uptr)
 {
     TOD_DATA *td;
     time_t delta;
-    uint32_t catchup_ticks;
+    uint32 catchup_ticks;
 
     if (!(uptr->flags & UNIT_ATT) || uptr->filebuf == NULL) {
         return;
@@ -305,7 +305,7 @@ static void tod_resync(UNIT *uptr)
     if (td->time > 0) {
         delta = time(NULL) - td->time;
         if (delta > MIN_DIFF && delta < MAX_DIFF) {
-            catchup_ticks = (uint32_t) delta * CLK_TPS;
+            catchup_ticks = (uint32) delta * CLK_TPS;
             sim_debug(EXECUTE_MSG, &tod_dev,
                       "Catching up with a delta of %ld seconds (%d ticks).\n",
                       delta, catchup_ticks);

